invenio_opendata/base/templates/get_started_CMS_2011.html

```diff
@@ -92,6 +92,9 @@ <h1 class="gen-title">Getting started with CMS 2011 data</h1>
       <li>
           In the <kbd>demoanalyzer_cfg.py</kbd> file, under the section to define the input data, select any primary dataset with muons from the 2011 data (such as <kbd>SingleMu</kbd> or <kbd>DoubleMu</kbd>).
       </li>
+      <li>
+          Download the list of validated runs from <a href="http://opendata.cern.ch/record/1001">http://opendata.cern.ch/record/1001</a> and, in the <kbd>demoanalyzer_cfg.py</kbd> file, replace <kbd>goodJSON = '/home/cms-opendata/CMSSW_4_2_8/src/Demo/DemoAnalyzer/datasets/Cert_136033-149442_7TeV_Apr21ReReco_Collisions10_JSON_v2.txt'</kbd> with <kbd>goodJSON = '/home/cms-opendata/CMSSW_5_3_32/src/Demo/DemoAnalyzer/datasets/Cert_160404-180252_7TeV_ReRecoNov08_Collisions11_JSON.txt'</kbd>.
```

Comment:
Yup, that would be good.  And, as the link text, you can put "from this record" or something similar?